api/v1alpha1/selfnoderemediationconfig_types.go

```diff
@@ -128,6 +129,19 @@ type SelfNodeRemediationConfigSpec struct {
 type SelfNodeRemediationConfigStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
+
+	// MinSafeTimeToAssumeNodeRebootedSeconds is the minimum value that can be assigned to SelfNodeRemediationConfigSpec.SafeTimeToAssumeNodeRebootedSeconds, it is calculated and assigned dynamically.
+	// +optional
+	// +kubebuilder:validation:Minimum=0
+	MinSafeTimeToAssumeNodeRebootedSeconds int `json:"minSafeTimeToAssumeNodeRebootedSeconds,omitempty"`
+
+	// +operator-sdk:csv:customresourcedefinitions:type=status,displayName="conditions",xDescriptors="urn:alm:descriptor:com.tectonic.ui:conditions"
```

Comment:
nit: for better readibilty, please start the comment with the field's description, and have all markers afterwards

Reply:
Yep, fair enough.
That's what I get for [copy pasting](https://github.com/medik8s/node-healthcheck-operator/blob/main/api/v1alpha1/nodehealthcheck_types.go#L218-L225) too quickly :see_no_evil: